core_backend/app/db/db_models.py

```diff
@@ -316,71 +321,260 @@ class ContentDB(Base):
     SQL Alchemy data model for content
     """
 
-    __tablename__ = "content"
+    __tablename__ = "contents"
 
     content_id: Mapped[int] = mapped_column(Integer, primary_key=True, nullable=False)
+
```

Comment:
Let's add content-level metadata?

Reply:
I'm strongly against that. Because so far we do not interact with contents but only with content_texts. Meaning that we are treating each content_texts as its own content and we only use contents to group them together. Adding contents will bring confusion as we will now have to treat them as two separate entities. Also, if we add it on contents we would have to start adding CRUD endpoints to contents as well which will bring confusion. But I am open to adding it if it is really important.

Reply:
We will be adding content-level information like tags down the line, but we can add it when we get to it!